script.js

```diff
@@ -0,0 +1,67 @@
+var choice = ["Rock", "Paper", "Scissors"];  
+choice.forEach(Function1);
+function Function1(item){
+var btn = document.createElement("BUTTON");   
+btn.classList.add("player_choice");                                
+document.getElementById("div1").appendChild(btn).textContent +=  item ;
+}
+let playerChoice ="";
+// console.log(playerChoice);
+ const choicesButton=document.querySelectorAll(".player_choice");
+ console.log(choicesButton);
+ choicesButton.forEach(button=>{
+    button.addEventListener("click",function(){
+    document.getElementsByClassName("player")[0].innerHTML="";
+    playerChoice=button.textContent;
+    console.log(playerChoice);
+    document.getElementsByClassName("player")[0].textContent += playerChoice ;
+    
+     })
+ })
+ 
+function Winner(player){
+
+    let computer= choice[Math.floor(Math.random() * choice.length)];
+    
+
+    
+    if (computer === player) {
+       alert ("It's a tie!");
+    }
```

Comment:
In this case, even if the result was a tie, it will also check the if statements below. and we don't want that.
to fix this problem, you can add `return` after this line:
```js
 if (computer === player) {
       alert ("It's a tie!");
    }
return;
```